src/tests/setupTests.js

```diff
@@ -13,6 +14,7 @@ global.afterAll(() => {
   server.close()
 })
 
-global.afterEach(() => {
+global.afterEach(async () => {
   queryCache.clear()
+  await flushAllPromises()
 })
```

Comment:
Ce changement a un effet en particulier sur les tests ?
Moins de warning ? 🤩

Reply:
Tu peux préciser dans la description de la PR pourquoi on fait ce changement ? 
Genre CI fails avec telle erreur (screenshot).
Cela nous aidera pour la traçabilité future :)

Reply:
Et modifier le nom du commit aussi : `FlushPromises after each test to fix CI failure "trying to import file after tests are finished"`

Reply:
je pense pas que ça a un effet sur les warning, du moins j'ai pas vérifié.
C'était pour fixer les jobs de CI qui échouent de façon aléatoire.